index.html

```diff
@@ -0,0 +1,102 @@
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css" integrity="sha512-5A8nwdMOWrSz20fDsjczgUidUBR8liPYU+WymTZP1lmY9G6Oc7HlZv156XqnsgNUzTyMefFTcsFH/tnJE/+xBg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+
+<body>
+  <nav id="navbar">
+    <div class="menu">
+      <a href="#">About</a>
+      <a href="#">Work</a>
+      <a href="#">Contact</a>
+    </div>
+  </nav>
+
+  <section id="welcome-section">
+    <h1>Hey I am Mimic</h1>
+    <p>a web developer</p>
+  </section>
+
+  <!-- project entry -->
+  <section id="projects">
+    <h2>This are some of my projects</h2>
+    <div class="project-container">
+      <div class="project-tile">
+        <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank" id="profile-link"><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg" alt="project01"></a>
+        <p>Tribute Page</p>
+      </div>
+      <div class="project-tile">
+        <a href="https://codepen.io/freeCodeCamp/full/wgGVVX" id="profile-link" target="_blank"><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png" alt="project02 a Calculator"></a>
+        <p>JavaScript Calculator</p>
+      </div>
+      <div class="project-tile">
+        <a href="https://codepen.io/freeCodeCamp/full/wgGVVX" id="profile-link" target="_blank"><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png" alt="project02 a Calculator"></a>
+        <p>JavaScript Calculator</p>
+      </div>
+      <div class="project-tile">
+        <a href="https://codepen.io/freeCodeCamp/full/zNqgVx" target="_blank" id="profile-link"><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg" alt="project01"></a>
+        <p>Tribute Page</p>
+      </div>
+      <div class="project-tile">
+        <a href="https://codepen.io/freeCodeCamp/full/wgGVVX" id="profile-link" target="_blank"><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png" alt="project02 a Calculator"></a>
+        <p>JavaScript Calculator</p>
+      </div>
+      <div class="project-tile">
+        <a href="https://codepen.io/freeCodeCamp/full/wgGVVX" id="profile-link" target="_blank"><img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png" alt="project02 a Calculator"></a>
+        <p>JavaScript Calculator</p>
+      </div>
+    </div>
+  </section>
+
+
+
+  <footer>
+    <div class="social-links">
+      <a href="#"><i class="fa fa-facebook-square"></i> Facebook</a>
+      <a href="#"><i class="fa fa-github"></i> Github</a>
+      <a href="#"><i class="fa fa-twitter"></i> Twitter</a>
+      <a href="#"><i class="fa fa-at"></i> Email</a>
+      <a href="#"><i class="fa fa-mobile"></i> Phone</a>
+    </div>
+
+    <div class="bottom-footer">
+      
+    </div>
+  </footer>
+
+
+
+
+
+
+
+
```

Comment:
Thank you for submitting your code for review. I noticed that there are several instances where there are excessive spaces between lines of code, which can detract from the professional appearance of the code. To improve the readability and maintainability of the code, I kindly recommend reducing the number of spaces between lines.

I appreciate your hard work on this project and look forward to seeing the updated code.

Reply:
- [OPTIONAL] You can consider using friendly emojis while making suggestions. Using friendly emojis makes your review sound friendly and calm. 👍